src/media/headset.ts

```diff
@@ -0,0 +1,71 @@
+import { Observable } from 'rxjs';
+import { SdkMedia } from "..";
+import HeadsetService, { ConsumedHeadsetEvents, VendorImplementation} from 'softphone-vendor-headsets';
+
+export class SdkHeadset {
```

Comment:
The style we generally follow projects we own is 
```
functionName (): ReturnType {

}
```
Will you pad all the function declaration parens with a space? Also please check fn's have a declared return type.

Reply:
I think most of the headsetLib fn's return promises right? I'd rather not swallow those promises, can we `return this.headsetLibrary.fn()` or at least await them?